spaceflights/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_processing/pipeline.py

```diff
@@ -1,8 +1,10 @@
-from kedro.pipeline import Pipeline, node
-from kedro.pipeline.modular_pipeline import pipeline
+from kedro.pipeline import Pipeline, node, pipeline
 
 from .nodes import create_model_input_table, preprocess_companies, preprocess_shuttles
 
+def create_pipeline(**kwargs) -> Pipeline:
+    return pipeline([])
+
```

Comment:
```suggestion
```
This looks redundant.

Reply:
Hi @noklam I committed your suggestion --  not sure I understand what the problem is here though!

Reply:
@stichbury There was 2 `create_pipeline` in the file

![image](https://user-images.githubusercontent.com/18221871/201162256-9545487a-1efe-49eb-9f23-e3a3ebbbf09c.png)